### DISCUSSION TITLE
Any way to have Renovate create an issue?

### DISCUSSION THREAD

**Discussion by ernievd** [renovatebot/renovate#14945] [Request Help]
Obviously I know that Renovate creates PR's/MR's.

Is there any way to also have it create an issue when it detects a dependency upgrade?

 BTW - I am using this in GitLab.

**Comment by rarkins** [comment]
See dependencyDashboard

**Reply by ernievd** [reply]
Besides that is there a way to have it create an issue the same way it creates an MR? 

We have a board that monitors issue and not MRs - It would be nice to have Renovate create an issue when it finds a new dependency version. We would apply labels to it like "to do" and "Dev ops" and it would then automatically come up on our issues board.

**Reply by rarkins** [reply]
No, there's no individual issues per update capability

**Reply by rarkins** [reply]
Or maybe.. you can customize dependencyDashboardTitle using templating to match the PR title
